### PR TITLE
e2e test additions

### PR DESCRIPTION
**Pull request by josojo**

### CODE REVIEW

File: contracts/DxMgnPool.sol [hunk 1]
```diff
@@ -138,7 +140,7 @@ contract DxMgnPool is Ownable {
         // Don't revert if wen can't claimSellerFunds
         address(dx).call(abi.encodeWithSignature("claimSellerFunds(address,address,address,uint256)", secondaryToken, depositToken, address(this), lastParticipatedAuctionIndex));
         mgnToken.unlockTokens();
-        totalDeposit = dx.balances(address(depositToken), address(this));
+        totalDeposit = dx.balances(address(depositToken), address(this)) + depositToken.balanceOf(address(this));
```
Comment (fleupold) [inline contracts/DxMgnPool.sol]: This fix should probably be a separate PR. Also, did you check if `dx.withdraw` reverts if we try to withdraw more than the balance inside `dx`? Maybe it'd be safer to only withdraw `dx.balances(address(depositToken), address(this))`

File: contracts/DxMgnPool.sol [hunk 2]
```diff
@@ -114,6 +114,9 @@ contract DxMgnPool is Ownable {
             depositToken.approve(address(dx), depositAmount);
             dx.deposit(address(depositToken), depositAmount);
         }
+        if (isDepositTokenTurn()) {
```
Comment (fleupold) [inline contracts/DxMgnPool.sol]: nit: no need to multiply with 2

Also maybe we can combine it with the if clause above?

**Comment by josojo** [comment]
The network changes only appeared after the merge of the other PR.... Cleaned it
